autofs-5.0.5 - fix simple bind without SASL support

From: Ian Kent <raven@themaw.net>

If autofs is compiled without SASL support simple bind authentication
fails to function.
---

 CHANGELOG             |    1 +
 modules/lookup_ldap.c |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)


--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -169,6 +169,7 @@
 - fix fix LDAP result leaks on error paths.
 - dont retry ldap connect if not required.
 - fix result null check in read_one_map().
+- fix simple bind without SASL support.
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/modules/lookup_ldap.c
+++ autofs-5.0.5/modules/lookup_ldap.c
@@ -1530,7 +1530,6 @@ int lookup_init(const char *mapfmt, int
 		}
 	}
 
-#ifdef WITH_SASL
 	/*
 	 *  First, check to see if a preferred authentication method was
 	 *  specified by the user.  parse_ldap_config will return error
@@ -1543,6 +1542,7 @@ int lookup_init(const char *mapfmt, int
 		return 1;
 	}
 
+#ifdef WITH_SASL
 	/* Init the sasl callbacks */
 	if (!autofs_sasl_client_init(LOGOPT_NONE)) {
 		error(LOGOPT_ANY, "failed to init sasl client");
